autopush/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "autopush"
-version = "1.57.0"
+version = "1.57.1"
```

Comment:
funny we haven't bumped autoendpoint's version yet. that might a whole separate discussion. but once we get it deployed and into testing we should revisit/figure its version

Reply:
Yeah, kinda wish there was a way to have the version shared among the crates. I updated the other two mostly out of habit, and just missed the endpoint.